csharp-basics/exercises/Collections/FlightPlanner/Program.cs

```diff
@@ -9,14 +9,76 @@ namespace FlightPlanner
 {
     class Program
     {
-        private const string Path = "../FlightPlanner/flights.txt";
+        private const string Path = @"E:\codelex\CODELEX_UZDEVUMI\06-2021\csharp-basics\exercises\Collections\FlightPlanner\flights.txt";
 
         private static void Main(string[] args)
         {
             var readText = File.ReadAllLines(Path);
-            foreach (var s in readText)
+            var flights = new Dictionary<string, string>();
+            var cities = new List<string>();
+            string startingCity;
+            string destination;
+
+            ConvertFlightsToDictionary(readText, flights);
+
+            Console.WriteLine("What would you like to do? ");
+            Console.WriteLine("To display list of cities press 1: ");
+            Console.WriteLine("To exit program press #");
+
+            bool runLoop = Console.ReadLine() == "1";
```

Comment:
uz kādu jautājumu atbild runLoop? shouldRunAgain?

Reply:
šeit bija daļa no uzdevuma, jo tur bija prasīts, lai programma sāktos ar
Console.WriteLine("To display list of cities press 1: ");
Console.WriteLine("To exit program press #");

Līdz ar to, programmas loop palaidīsies tikai tad, ja lietotājs ievadīs 1

Reply:
Es par bool name runāju. Bool name ir jābū jautājuma formā